src/mentions/SingleMention.ts

```diff
@@ -151,4 +151,11 @@ export class SingleMention extends Mention {
 
 		return embed;
 	}
+	private shorten( input: string ): string {
```

Comment:
Some name like `ensureLength` might be more descriptive.

Reply:
I have changed this, thanks!